libraries/user.rb

```diff
@@ -0,0 +1,70 @@
+class Chef
+  # Chef Resource for declaring a user and group for Elasticsearch
+  class Resource::ElasticsearchUser < Resource
+    include Poise
+
+    actions(:create, :remove)
```

Comment:
I would recommend leaving them, since we're emulating a style that other users and chef community people will recognize. It seems like they are most often omitted in LWRPs and not omitted in HWRPs. I don't feel that strongly about it though, if everyone else feels differently. I just want to be sure what we write is approachable enough that we can get good contributions from the community.